homeassistant/components/mqtt/__init__.py

```diff
@@ -171,6 +175,10 @@
     Platform.VACUUM,
 ]
 
+PLATFORMS_YAML_SETUP: list[Platform] = [
+    Platform.FAN,
+    Platform.SENSOR,
```

Comment:
The PR includes changes in fan + light, not fan + sensor.

Reply:
Ah that is hence the light platform did not initialize 🥇